src/main/java/me/mrCookieSlime/Slimefun/SlimefunPlugin.java

```diff
@@ -391,21 +391,15 @@ public void onDisable() {
             }
         });
 
-        for (World world : Bukkit.getWorlds()) {
+        // Save all registered Worlds
+        getRegistry().getWorlds().forEach((world, storage) -> {
```

Comment:
Please don't use .forEach(...), it can have quite a negative impact on performance. It is advised to use the standard for loop format instead.

Reply:
Normally it should be done only once when shutting down the server without any players already but I see the point. Agreed, should be resolved now.